kn/fotos/media/fotos.js

```diff
@@ -405,16 +478,36 @@
       maxWidth -= 200;
     }
     if (width > maxWidth) {
-        height *= maxWidth/width;
-        width  *= maxWidth/width;
+      height *= maxWidth/width;
+      width  *= maxWidth/width;
     }
     if (height > maxHeight) {
-        width  *= maxHeight/height;
-        height *= maxHeight/height;
+      width  *= maxHeight/height;
+      height *= maxHeight/height;
+    }
+    // keep up to date with entities.py
```

Comment:
Het is onderhoudbaarder om op maar één plek dezelfde logica te hebben.  Kun je na een rotatie niet gewoon de pagina "verversen"?